README.md

```diff
@@ -523,6 +523,7 @@ If you are using ProGuard you might need to add the following options:
 -keepclasseswithmembers class * {
     @com.squareup.moshi.* <methods>;
 }
+-keep @com.squareup.moshi.JsonQualifier class *
```

Comment:
Seems like that is the case. I have my annotations in their own class files though, let me see if it matters to have them as inner classes of the adapter or something else like that.